src/backend/columnar/columnar_reader.c

```diff
@@ -285,11 +328,13 @@ ColumnarReadRowByRowNumber(Relation relation, uint64 rowNumber,
  * Errors out if no such row exists in the stripe being read.
  */
 static void
-ReadStripeRowByRowNumber(StripeReadState *stripeReadState,
-						 StripeMetadata *stripeMetadata,
+ReadStripeRowByRowNumber(ColumnarReadState *readState,
```

Comment:
I'm curious why the signature is changed, it seems like you're passing the larger struct and just extracting the same two parts of it?

Reply:
yes, this is the reason